client/dashboard/utils/user.ts

```diff
@@ -0,0 +1,9 @@
+import { UserFlags } from '../data/me';
+import type { User } from '../data/types';
+
+export const userHasFlag = ( user: User, flag: `${ UserFlags }` ) => {
+	if ( ! user ) {
```

Comment:
TS tells us that `user` is always defined here.

And we can trust the TS. We see here that the components don't render until `user` is defined.

https://github.com/Automattic/wp-calypso/blob/7606df1fd88e4c06214bd30ada6fce771099e1fa/client/dashboard/app/auth/index.tsx#L44-L46

So the undefined check isn't needed.

Also now that this functions is just a wrapper around `Array.prototype.includes` I'm not sure we should have this whole function.

Reply:
This was for debugging. I forgot to remove this...